### DISCUSSION TITLE
octavus changes

### DISCUSSION THREAD

**Discussion by cloudziq** [redeclipse/base#1200] [Maps and Content Showcase]
changes:
fixed eagles positioning, also they fly above the entire map now
added new blendmaps + changes to some blend layered textures
nicer looking glass
environment vars changed a bit
added sounds and a bit of geometry to doors
added small bushes around the map

in overall, map contrasting much better with surrounding environment, its not so clean outside anymore like structures were built day ago
before:
![20200214131907](https://user-images.githubusercontent.com/13541407/76028757-86eef700-5f33-11ea-9754-197247168425.jpg)

now:
![20200214132043](https://user-images.githubusercontent.com/13541407/76028797-9a9a5d80-5f33-11ea-8fca-6b7a3ec1f848.jpg)

[octavus.zip](https://github.com/redeclipse/world/files/4295280/octavus.zip)

**Comment by Jigoku** [comment]
Those changes look nice, but i think it's lacking a bit of colour. Not sure how everything is very grey (feels like a different map). I think it might be best to keep the orange trims in place; 

eg; here's a quick vcolour/replace

![20200305222342](https://user-images.githubusercontent.com/1535179/76031839-70936c00-5f30-11ea-8fed-fb64fcec2630.png)

**Comment by cloudziq** [comment]
earlier were too red, now too grey?  ;p  If you want, you can change this and send me the mpz and cfg so I can do fast commit.  Or you want me to change this? ;>

**Comment by Jigoku** [comment]
Hmm i think i remember saying the lighting (probably ambient) made the white buildings textures look a bit too plain. One idea was to use a mild normal map from another texture, like panels etc. 

But your lighting changes there help with that, looks good. The slightly darker surface makes the shadows look more natural. I just think it might be a good idea to keep the orange trims in there as for the maps character.

Maybe best to see what others think :thinking:

**Comment by Jigoku** [comment]
Also; i think i used;

`/vcolour 0.75 0.75 0.75` on the grey/white outer walls
and
`/vcolour 1 0.7 0.2` on the orange trims

in that quick test above

**Comment by cloudziq** [comment]
ok I will do what you suggest :)

**Comment by cloudziq** [comment]
Latest remix of Octavus:
  -a bit expanded waypoints, fixed in many places
  -added railed + standard cams to cover as much as possible
  -changed lighting to have better contrast, also with a more different variant 'look'
  -added sounds to doors
  -changes to eagles so they fly in a wider area 
  -smoothen out geometry around certain objects
  -changed texture blendmapping
[octavus.zip](https://github.com/redeclipse/world/files/4555791/octavus.zip)

**Comment by Jigoku** [comment]
I like it, looks great overall, but i still think it needs the orange trims left in there for some contrast, otherwise everything just becomes a bit too grey and ends up clashing with the sand. As for it being an outpost on a desert planet, i just don't think it blends in with the environment well without the orange parts.
